admin.widget.class.php

```diff
@@ -133,10 +142,16 @@ public function wpsp_admin_page()
         <?php
     }
 
+    public function admin_page_header($arg) {
+      $image_url = $this->asset_url('assets/logo-40.png');
+      printf('<div class="sp-heading"><img src="%s" alt="SparkPost"> &nbsp;&nbsp; <h2>%s</h2></div>', $image_url, 'SparkPost');
+    }
+
     public function admin_page_init()
     {
+        wp_register_style('sp-admin-css', $this->asset_url( 'assets/styles.css'));
         register_setting('sp_settings_group', 'sp_settings', array($this, 'sanitize'));
-        add_settings_section('general', 'SparkPost Settings', null, 'sp-options');
+        add_settings_section('general', null, array($this, 'admin_page_header'), 'sp-options');
```

Comment:
Yeah, i found it nothing breaks too. but i think it's better to supply empty string (unless it generates empty `h2` tag with tag)